src/functions.ts

```diff
@@ -970,3 +970,9 @@ export async function setLocalMobileUxConfiguration(
     }
   }, 'setLocalMobileUxConfiguration')();
 }
+
+export async function getNativeSdkVersion(): Promise<string> {
+  return Logger.traceSdkMethod(async () => {
+    return await StripeTerminalSdk.getNativeSdkVersion();
```

Comment:
try / catch block?

Reply:
Receive the const will not cause exception but we can align the usage and prevent future break, will add in following commit.

Reply:
I found few different approaches, would like to hear from your opinion:

- Wrap the result with error like `confirmRefund()` does below to consistently return same type to caller:
```
export type ConfirmRefundResultType = {
  refund?: Refund.Props;
  error?: StripeError;
};
```
- Return string when success, return error object when fail like `getOfflineStatus()` does
```
export type OfflineStatus = {
  sdk: OfflineStatusDetails;
  reader?: OfflineStatusDetails;
};
```
- Return empty string or `NA` when error happen to keep using string type

Reply:
i'm peaceful with returning an empty string. i don't expect errors to happen here but juuuuuust in case we don't break